src/world/climate.h

```diff
@@ -26,20 +26,22 @@ enum {
 	CLIMATE_COLD
 };
 
-enum{
+enum {
 	WEATHER_SUNNY,
 	WEATHER_PARTIALLY_CLOUDY,
 	WEATHER_CLOUDY,
 	WEATHER_RAIN,
 	WEATHER_HEAVY_RAIN,
-	WEATHER_THUNDER
+	WEATHER_THUNDER,
+	WEATHER_SNOW
```

Comment:
Before you can do this, you have to check what happens when you load a saved game that is snowing in vanilla RCT2. It may crash the original game.